operator/src/scripts/multi-cloud-deployment/deploy-gke.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
```

Comment:
We should keep this script folder clean. Only the scripts which are used/run the the operator itself or we need inside the host machine. 

All other setup scripts for different environment should go under `documentdb-playground/` specific folder.